app/controllers/admin/loans_controller.rb

```diff
@@ -1,5 +1,17 @@
 class Admin::LoansController < Admin::AdminController
   def index
     @loans_grid = initialize_grid(Loan, include: [:division, :organization])
+
+    # TODO:
+      # Only load coops/organizations with loans into loan grid
+      # Filter in view should only show coops with loans
+
+    # TODO:
```

Comment:
There is some wonkiness related to name. "Project ...." is loan.name and "Loan ...." is loan[:name]. I don't know why these are not the same. This is seems to be an issue with the model and Factory Girl.